### DISCUSSION TITLE
Cannot switch from anonymous user in Kibana, and no so-elastic-auth

### DISCUSSION THREAD

**Discussion by abesinger** [Security-Onion-Solutions/securityonion#4431] [Unsupported Versions] [answered]
I want to second the complaint in this previous post, I too do not have so-elastic-auth installed. I' m using a distributed install, version 2.3.51, installed from the Centos ISO: _Originally posted by @seg1b in https://github.com/Security-Onion-Solutions/securityonion/discussions/3304_

Furthermore, when I open the Kibana page and enter my email addr and pw, they get accepted, but when I click on the user icon in the upper right hand corner, it shows me logged in as anonymous_user.  When I then try the "Log in" link, I get a page that says "Successfully logged out" with a "Log in" button.  When I click that button, I'm automatically taken back to Kibana as anonymous_user. 

And to repeat what I said above, I cannot find so-elastic-auth anywhere on my Manager node.

**Comment by TOoSmOotH** [comment, marked as answer]
https://github.com/Security-Onion-Solutions/securityonion/discussions/4434
